xbmc/windowing/WinSystem.h

```diff
@@ -84,6 +84,13 @@ class CWinSystemBase
   //the number of presentation buffers
   virtual int NoOfBuffers();
 
+  /*!
+   * \brief Forces the window to fullscreen provided the window resolution
+   * \param resInfo - the resolution info
+   *
+  */
```

Comment:
The end of this doxy is a little funky, you could remove the empty line and align the closer.